packages/core/src/components/TextField/TextField.tsx

```diff
@@ -171,13 +176,25 @@ const TextField: VibeComponent<TextFieldProps, unknown> & {
       [onChange, isRequiredAndEmpty]
     );
 
-    const { inputValue, onEventChanged, clearValue } = useDebounceEvent({
+    const {
+      inputValue: uncontrolledInput,
```

Comment:
why?

Reply:
The thing is that the debounce has a state, this is the root cause of the issue. I can add debounce but I think that if the user wants to control the input they should have the full flexibility